lib/checks/checks.go

```diff
@@ -256,6 +255,32 @@ func FormatFailedChecks(failed []*agentpb.Probe) string {
 	return buf.String()
 }
 
+// OverrideDockerConfig updates given config with values from overrideConfig where necessary
```

Comment:
Feels like something's missing b/w "updates" and "given".

Reply:
What's missing? The method **updates** given **config** (hence passed by reference)